### ISSUE TITLE
Redirected gwt artifacts cause a NPE when generating modules

### ISSUE THREAD

**Issue by niloc132** [tbroyer/gwt-maven-plugin#150]
Plugin version 1.0.0
Maven version 3.8.3

Steps to reproduce: 
 * Create a gwt-app that uses this plugin, and use the default settings (skipModule=false)
 * Add the maven repo https://repo.vertispan.com/gwt-groupid-migration-test/ (releases only, no snapshots), and specify gwt version `2.10.0-new-groupid` with the traditional groupId, com.google.gwt throughout the project.
 * Try to build such that the `gwt:generate-module` goal will run, such as `mvn verify`.

Expected results: Project will build and run tests correctly.

Actual results:
```
Caused by: java.lang.NullPointerException
    at net.ltgt.gwt.maven.GenerateModuleMojo.generateInheritsFromDependencies (GenerateModuleMojo.java:251)
    at net.ltgt.gwt.maven.GenerateModuleMojo.execute (GenerateModuleMojo.java:180)
```

Workarounds:
 * Specify skipModule=true in the configuration and manually construct a gwt module file
 * Alternatively, in this sample it appears to work to keep the pom import, but update the other gwt artifacts with groupid=`org.gwtproject`. Very quick tests indicate that this still works correctly even if com.google.gwt is picked up transitively: maven resolves this issue before it gets to the mojo, and/or the GenerateModuleMojo doesn't pick up transitive dependencies.

**Comment by tbroyer** [comment]
Fwiw, using Java 17:
```
Caused by: java.lang.NullPointerException: Cannot invoke "java.io.File.toURI()" because the return value of "org.apache.maven.artifact.Artifact.getFile()" is null
    at net.ltgt.gwt.maven.GenerateModuleMojo.generateInheritsFromDependencies (GenerateModuleMojo.java:251)
    at net.ltgt.gwt.maven.GenerateModuleMojo.execute (GenerateModuleMojo.java:180)
```

This is actually quite likely to precipitate switching `generateInheritsFromDependencies` to being `false` by default (and/or removed entirely), but let's release a quick fix for now, ignoring artifacts without files (which is the case for relocated artifacts).

**Comment by tbroyer** [comment]
This should be in the latest snapshot, could you confirm it works as expected (without error) on your side?

**Comment by niloc132** [comment]
Confirmed, sorry for the delay.

**Comment by tbroyer** [comment]
Released as 1.0.1.
